src/components/splash.jsx

```diff
@@ -45,10 +47,53 @@ export default function Splash({
     </form>
   );
 
+  const learnMore = (
+    <Collapsible trigger="Learn more">
+      <h3>General</h3>
+      <p>Make sure that you have the right privileges to list apps or view the app.</p>
+      <h3>Engine running in Docker container</h3>
+      <p>
+        For connecting to an Engine running in a Docker container with a mounted data volume containing the app and starting
+        engine with
+        {' '}
+        <strong>DocumentDirectory=&lt;data-folder&gt;</strong>
+, the websocket URL will
+        be
+        {' '}
+        <strong>ws://&lt;host&gt;:&lt;port&gt;/&lt;data-folder/&lt;app&gt;</strong>
+. E.g.
+        {' '}
+        <strong>ws://localhost:9076/data/my-excellent-app</strong>
+.
+      </p>
+      <h3>Qlik Sense Enterprise on Windows</h3>
+      <p>
+        For connecting to a Qlik Sense Enterprise on Windows using default virtual proxy, the websocket URL will
+        be
+        {' '}
+        <strong>wss://&lt;sense-host.com&gt;/app/&lt;app-GUID&gt;</strong>
+. If using a custom virtual proxy the
+        websocket URL will need to reflect this. Also note that for the Sense Proxy to allow sessions from catwalk,
+        catwalk needs to be whitelisted in Qlik Sense Enterprise.
+      </p>
+      <p>
+        Make sure that you are logged in to Qlik Sense in another browser tab/window. This is needed in order for catwalk
```

Comment:
Maybe clarify this a bit? catwalk doesn't use the cookie, the browser will use the session cookie directly towards their deployment, nothing passes through the catwalk logic :)

Reply:
yepp, true!